### ISSUE TITLE
Multiple Click Listeners

### ISSUE THREAD

**Issue by cyono** [dequelabs/axe-android#44]
To be filled out later.

A description of the rule that would show up in the User Interface. Should be 10 - 20 words.

**Affected Technology**: [TalkBack, Switch Control, Magnification, ...]
**Type**: [WCAG, Platform, Best Practice]
**Success Criteria**: A link to the related SC if applicable.

**Note**: This is a template. Please delete (Required/Optional) annotations and any **Not Filled** in **Optional** fields before submitting... otherwise we will make fun of you.

## User Story (Required)

A short summary or introduction to the users you are concerned about.

### Inaccessible (Required)

1. An ordered list of steps from the users point of view.
2. Should be a numbered list.

A short description of why the behavior you've outlined above is inaccessible.

### Accessible (Required)

A similar list of behaviors or short description as to how the inaccessible experience 
should be improved. 

## Technical Defense (Optional)

Defend your proposal. Why is this a hardcore requirement that every App in the world 
should adhere to? 

**Note**: This section is **very** important when there is **no** associated WCAG Success Criteria 
or when the associated Success Criteria is not directly applicable. 

### Screenshot (Optional)

Annotate a screenshot demoing a **Violation** of your proposed rule.

### Google Play Example (Optional)
A link to a Google Play App that violates this rule

1. Instructions on how to get to the view.
2. Should be a numbered list.

### Contextual Information (Optional)

- Attachment: results.axe of an AxeView or AxeEventStream that would violate this rule.
- Screenshot: If it makes more sense to attach rather than include inline.
